.travis.yml

```diff
@@ -1,12 +1,30 @@
+dist: trusty
 sudo: false
 language: elm
+addons:
+  chrome: stable
+
+services:
+  - docker
 
 cache:
   directories: # so subsequent builds run faster
     - elm-stuff
     - $HOME/.elm
     - $HOME/.npm # https://stackoverflow.com/a/42523517/1148249
 
+before_install:
+  - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
+
 script:
   - elm-format --validate .
-  - npm run test:client
+  - npm run test:ci
+  - docker build -t "nakadi/nakadi-ui:${TRAVIS_BRANCH}-${TRAVIS_BUILD_NUMBER}" .
```

Comment:
does Nakadi UI support compatibility with Nakadi in anyway at the moment?

Reply:
Nakadi API stable enough so I don't see the need for versions synchronization for now.
But we can add it in the future.

Reply:
I am fine with it :) just wanted to highlight that you have to use always latest version of Nakadi UI and Nakadi together